swift-sdk/Internal/EmptyEmbeddedManager.swift

```diff
@@ -5,6 +5,9 @@
 import Foundation
 
 class EmptyEmbeddedManager: IterableEmbeddedManagerProtocol {
```

Comment:
Whats the purpose of EmptyEmbeddedManager. Not sure why we have that for InAppManager as well!